### DISCUSSION TITLE
error in log's No matching routes for HEAD /.

### DISCUSSION THREAD

**Discussion by GieGit** [dani-garcia/vaultwarden#3303] [Q&A]
### Subject of the issue
I find the following message in the log
[2023-03-02 08:59:14.617][_][ERROR] No matching routes for HEAD /.

### Deployment environment

* vaultwarden version:

* Install method: Docker

* Clients used:  iOS, 

* Reverse proxy and version: Nginx-Proxy-Manager 

* Other relevant details:

**Comment by BlackDex** [comment]
Something or someone is trying to access your sure using a head request. Which is not supported by Vaultwarden.
Mostly this is done by remote uptime checking services like UptimeRobot or StatusCake.

I had something similar, and pointed it to the `/alive` endpoint and made sure nginx uses GET for that specific endpoint.

**Reply by GieGit** [reply]
thanks it was my problem, i use a checkup service since 3 day. if i stop this services the logfile is clean.

**Comment by jjlin** [comment]
The HEAD request is actually working fine; these are spurious error messages due to https://github.com/SergioBenitez/Rocket/issues/1098.

PR #3307 should address this issue.
